### ISSUE TITLE
ceedling 0.32 supports running single test case

### ISSUE THREAD

**Issue by u6bkep** [numaru/vscode-ceedling-test-adapter#115]
ceedling 0.32 release candidate supports running a single test case from a test file. it would be great if ceedling test adapter supported this as well.
[Feature - add functionality to run single or multiple tests from one or multiple test files](https://github.com/ThrowTheSwitch/Ceedling/pull/736)

**Comment by jnz86** [comment]
Agreed. Although It's been quite awhile and Ceedling 0.32 still isn't released.
